gnosis/safe-react
# Chore: coded exceptions registry and utils

This is based on the proposal from #2161.

We added:
* A registry of errors by code with a few sample errors
* Utilities to create errors with codes and log them depending on the env
* Unit tests
* A single example how to use it in `fetchTokens.ts`

## How to test
* Go to the assets page
* Make sure the initial assets are displayed
* Disable the network in the devtools
* Wait for a request for the tokens to fail
* Observe an error in the console

<img width="437" alt="Screenshot 2021-05-18 at 16 49 05" src="https://user-images.githubusercontent.com/381895/118681624-02b72580-b800-11eb-80d0-bfda144c0b90.png">

Closes #2161
